<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDlgLogin</class>
 <widget class="QDialog" name="QDlgLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/imgs/屏幕快照 2019-02-12 下午7.09.26.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="editUser">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>420</y>
     <width>136</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>user</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>39</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editPSWD">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>420</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>12345</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkSaveUserName">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>420</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>420</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>420</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>QDlgLogin</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>576</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
